app/src/main/java/com/github/h3lp3rs/h3lp/database/repositories/EmergencyInfoRepository.kt

```diff
@@ -6,6 +6,7 @@ import java.util.concurrent.CompletableFuture
 
 /**
  * Repository for emergency information communication with the database
+ * @param database The database from which to create the repository
  */
 class EmergencyInfoRepository(private val database: Database) : Repository<EmergencyInformation>{
```

Comment:
By the way, this class is not really useful. It could be removed when I think about it. It doesn't offer a lot of useful database wrapping.

Reply:
This can be discussed at our next meeting but good point indeed